sys/windows/windmain.c

```
@@ -1350,7 +1371,16 @@ void
 stdio_raw_print(const char *str)
 {
     if (str)
+#if 0 /*JP:T*/
         fprintf(stdout, "%s\n", str);
+#else
+/*        jputs(str);*/
+        {
+            FILE *f = fopen("z:/ll2.txt", "a");
+            fprintf(f, "%s\n", str);
```

Cppcheck (2.21.0)

[warning] If resource allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfResources)